client/components/ovh-task-follow/modal/ovh-task-follow-modal.controller.js

```diff
@@ -0,0 +1,29 @@
+angular.module("managerApp").controller("ovhTaskFollowModalCtrl", function ($scope, $uibModalInstance) {
```

Comment:
When I go on the manager I see this message => 

×
Nous vous informons que nous rencontrons actuellement des dysfonctionnements pouvant impacter votre service. Nos équipes techniques sont actuellement en cours d'intervention, les services sont rétablis au fur et à mesure, vous pouvez suivre l'évolution de la tâche sur http://travaux.ovh.net/?do=details&id=28256. Nous vous prions de nous excuser pour ces désagréments.

when I click on the link, I get redirect to a travaux task, which is good.

However I see no follow buttons... thereforce I'm wondering what is the utility this modal?